src/js/facility-locator/components/SearchResult.jsx

```diff
@@ -11,26 +11,40 @@ class SearchResult extends Component {
       `${address.city}, ${address.state} ${address.zip}-${address.zip4}`
     ];
 
+    /* eslint-disable camelcase */
+    const facilityTypes = {
+      va_health_facility: 'Health',
+      va_cemetary: 'Cemetary',
+      va_benefits_facility: 'Benefits',
+    };
+    /* eslint-enable camelcase */
+
     return (
       <div className="facility-result">
         <Link to={`facilities/facility/${facility.id}`}>
           <h5>{name}</h5>
         </Link>
-        <strong>Facility type: {facility.type}</strong>
+        <span>Facility type: <strong>{facilityTypes[facility.type]}</strong></span>
         <p>
           {addressString[0]}<br/>
           {addressString[1]}
         </p>
-        <span style={{ marginRight: '1rem' }}>
-          <a href={`tel:${phone.main}`}>
-            <i className="fa fa-phone"/> {phone.main}
-          </a>
-        </span>
-        <span>
-          <a href={`https://maps.google.com?saddr=Current+Location&daddr=${addressString.join(' ')}`} target="_blank">
-            <i className="fa fa-map"/> Directions
-          </a>
-        </span>
+        <div className="row">
+          <div className="columns small-12 large-6" style={{ paddingRight: 0 }}>
+            <span>
+              <a href={`tel:${phone.main}`}>
+                <i className="fa fa-phone" style={{ marginRight: '0.5rem' }}/> {phone.main}
+              </a>
+            </span>
+          </div>
+          <div className="columns small-12 large-6" style={{ paddingRight: 0 }}>
+            <span>
+              <a href={`https://maps.google.com?saddr=Current+Location&daddr=${addressString.join(' ')}`} target="_blank">
```

Comment:
Will React do the necessary escaping on this string? If it does, will `Current+Location` end up being double escaped? I always get itchy when I see string interpolation into a URL because I feel like there's always going to be weird data that screws it up (like an apostrophe or something).